modules/tinymce/src/core/main/ts/api/commands/SelectionCommands.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import * as NodeType from '../../dom/NodeType';
+import type Editor from '../Editor';
```

Comment:
```suggestion
import Editor from '../Editor';
```

Reply:
Curious why we can't use a type import here? I think all type only imports should be using that since we are not really importing code here and in theory this is a circular reference since Editor imports the command module and the commands import the Editor module. Making it a type only import makes that explicit that no side effects or code are being imported only the type.

Reply:
I explained this here: https://github.com/tinymce/tinymce/pull/7572#discussion_r795337288